package.json

```diff
@@ -4,7 +4,9 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "start": "webpack-dev-server --config ./webpack.config.js --mode development",
+    "start": "npm run build",
+    "start:dev": "webpack-dev-server --config ./webpack/webpack.dev.js --mode development",
```

Comment:
I don't think you still need to specify `--mode development` flag in the `start:dev`, since you have created `webpack/webpack.dev.js` mode property is specified there already.

Reply:
Done